### PR TITLE
Adding surveys for LEVANTE users

### PR DESCRIPTION
**Pull request by Zio-4**
- For child (students) users, surveys are shown before the last task which will be an external MEFs task.
- For parent and teacher users, surveys will be the only thing shown to them.

### CODE REVIEW

File: src/components/auth/SignIn.vue [hunk 1]
```diff
@@ -70,6 +70,8 @@ import { useAuthStore } from '@/store/auth';
 
 const authStore = useAuthStore();
 const { roarfirekit } = storeToRefs(authStore);
+const isLevante = import.meta.env.MODE === 'LEVANTE';
+const signInBtnLabel = isLevante ? 'Sign in' : "Go! →"
```
Comment (Zio-4) [inline src/components/auth/SignIn.vue:74]: This is now outdated due to i18n

File: src/pages/HomeParticipant.vue [hunk 2]
```diff
@@ -136,6 +136,14 @@ const {
   staleTime: 5 * 60 * 1000,
 });
 
+const { data: surveyResponsesData } = useQuery({
```
Comment (Zio-4) [inline src/pages/HomeParticipant.vue:139]: This query should only run for LEVANTE instance

Comment (Zio-4) [reply src/pages/HomeParticipant.vue:139]: Add env variable logic in enabled field

File: src/pages/LEVANTE/LEVANTE_Teacher_Survey.json
Comment (Zio-4) [inline src/pages/LEVANTE/LEVANTE_Teacher_Survey.json:1]: Remove from tracking

File: src/pages/LEVANTE/UserSurvey.vue [hunk 3]
```diff
@@ -0,0 +1,92 @@
+<script setup>
```
Comment (Zio-4) [inline src/pages/LEVANTE/UserSurvey.vue:1]: Add route guard to this page

File: src/pages/SignIn.vue [hunk 4]
```diff
@@ -48,13 +49,21 @@ import { isMobileBrowser } from '@/helpers';
...
 const incorrect = ref(false);
+const isLevante = import.meta.env.MODE === 'LEVANTE';
 const authStore = useAuthStore();
 const router = useRouter();
 
 const { spinner, authFromClever } = storeToRefs(authStore);
 
 authStore.$subscribe(() => {
   if (authStore.uid) {
+    if (authStore.userData) {
```
Comment (Zio-4) [inline src/pages/SignIn.vue:60]: add && condition to check for LEVANTE project

File: src/pages/SignIn.vue [hunk 5]
```diff
@@ -48,13 +49,21 @@ import { isMobileBrowser } from '@/helpers';
...
+      if (toRaw(authStore.userData.userType.toLowerCase()) === 'parent' || toRaw(authStore.userData.userType.toLowerCase()) === 'teacher') {
```
Comment (Zio-4) [inline src/pages/SignIn.vue:61]: Move this logic into HomeSelector. Make survey page a component.

File: src/router/index.js [hunk 6]
```diff
@@ -226,6 +232,12 @@ const routes = [
     component: () => import('../pages/ScoreReport.vue'),
     meta: { pageTitle: 'View Scores', requireAdmin: true },
   },
+  {
```
Comment (Zio-4) [inline src/router/index.js:235]: Add route guard for LEVANTE and to block student from taking survey before completing rest of tasks

**Review by kellyel — APPROVED** [review]

**Pull request merged**